ownCloud/ownCloud/Bookmarks/Connection Issues /IssuesViewController.swift

```diff
@@ -0,0 +1,217 @@
+        bottomContainer.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        bottomContainer.contentHuggingPriority(for: .vertical)
+
+        issuesTableView.translatesAutoresizingMaskIntoConstraints = false
+        issuesTableView.layer.cornerRadius = 10
+        issuesTableView.backgroundColor = UIColor(hex: 0xF2F2F2)
+        issuesTableView.bounces = false
+        issuesTableView.showsVerticalScrollIndicator = false
+
+        self.view.addSubview(issuesTableView)
+
+        issuesTableView.bottomAnchor.constraint(equalTo: bottomContainer.topAnchor, constant: -20).isActive = true
+        issuesTableView.leftAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leftAnchor, constant: 20).isActive = true
+        issuesTableView.rightAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.rightAnchor, constant: -20).isActive = true
+        issuesTableView.topAnchor.constraint(greaterThanOrEqualTo: self.view.safeAreaLayoutGuide.topAnchor, constant: 20).isActive = true
+        tableHeighConstraint = issuesTableView.heightAnchor.constraint(equalToConstant: issuesTableView.contentSize.height)
+        tableHeighConstraint?.isActive = true
+
+        issuesTableView.tableHeaderView?.backgroundColor = UIColor.white
+    }
+
+    override func viewWillLayoutSubviews() {
+        super.updateViewConstraints()
+        self.tableHeighConstraint?.constant =  self.issuesTableView.contentSize.height
+    }
+
+    func addBottomContainerElements(elements: [UIView]) {
+        for element in elements {
+            self.bottomContainer.addSubview(element)
+        }
+    }
+
+    @objc func dismissView() {
+        self.dismiss(animated: true, completion: nil)
+    }
+}
+
+extension IssuesViewController: UITableViewDelegate {
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return UITableViewAutomaticDimension
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+
+        guard self.headerTitle != nil else {
+            return 0
+        }
+
+        return 40
+    }
+}
+
+extension IssuesViewController: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let cell = UITableViewCell(style: .default, reuseIdentifier: nil)
+        cell.backgroundColor = .white
+        cell.textLabel?.attributedText = NSAttributedString(string: self.headerTitle!, attributes: [.font : UIFont.systemFont(ofSize: 20, weight: .semibold)])
+
+        let separatorView: UIView = UIView()
+        separatorView.translatesAutoresizingMaskIntoConstraints = false
+        separatorView.backgroundColor = UIColor(hex: 0x04040F)
+        cell.addSubview(separatorView)
+        separatorView.heightAnchor.constraint(equalToConstant: 0.25).isActive = true
```

Comment:
Even assuming one point being represented by 3 x 3 pixels, the minimum size for this view to be reliably visible would be 0.33, but still be too small for 2 x 2 pixel representations.

To always have this separator visible, on all devices, please make this 1 point.

(In this screenshot, it's missing:)
![bildschirmfoto 2018-03-22 um 23 30 49](https://user-images.githubusercontent.com/639669/37802027-17135ff8-2e29-11e8-9949-cf13b37f3010.png)

Reply:
Done!